bin.src/createSkyFile.py

```diff
@@ -85,7 +85,7 @@ def createSkyFile(
         # for the sources in this script. We just want to make sure
         # we get all sources within the radius and save them to a file.
         refCatInterface = RefCatalogInterface(ra, dec, 0.0)
-        shardIds = refCatInterface.getShardIds(radius=radius)
+        shardIds = refCatInterface.getHtmIds(radius=radius)
```

Comment:
I think this might be: `htmIds = refCatInterface.getHtmIds(radius=radius)`. Right?

Reply:
Yes, I will make this change to make it consistent.